send_telegram.py

```diff
@@ -0,0 +1,40 @@
+import requests
+import os
+from logger import logger
+from load_config import *
+
+config = load_config('config.yml')
+
+send_success = False
+
+if "BOT_TOKEN" in config['TELEGRAM'] and "BOT_CHAT_ID" in config['TELEGRAM']:
+   bot_token = config['TELEGRAM']['BOT_TOKEN']
+   bot_chatID = str(config['TELEGRAM']['BOT_CHAT_ID'])
+   send_success = True
```

Comment:
Maybe rename this one? It sounds like something was send successfully but all that happened so far is reading the id and token right?

Reply:
yeah this is confusing, gonna change